lib/cli/line-processor.js

```diff
@@ -68,11 +68,11 @@ module.exports = class LineProcessor {
     const [host, portString] = utils.split(authority, ':', 2, true);
     if (!host) {
       return callback(reportMissingArgument('Host'));
-    } else if (!portString) {
+    } else if (scheme !== 'ipc' && !portString) {
```

Comment:
Maybe it's better to put everything under `!== 'ipc'` case instead of checking it 2 times, it may be cleaner in my opinion.